apps/fp-frontend-app/src/behandlingsupport/melding/MeldingIndex.tsx

```diff
@@ -30,6 +30,9 @@ interface Props {
   hentOgSettBehandling: () => void;
 }
 
+const finnFrist = (behandling : BehandlingAppKontekst) =>
+  behandling.behandlingPaaVent ? behandling.fristBehandlingPåVent || behandling.fristBehandlingPaaVent : undefined;
```

Comment:
Treng ein å sjekka både versjonen med aa og å? (Kanskje grunna fptilbake?)

Det er forsovidt ikkje noko problem her, sidan fristane er datoar, men greit å vera klar over bruken av || vs ??. Sonar klagar ofte om ein brukar ||, og det er fort gjort å gå på ein smell om ein sjekkar mot tallverdiar (For || er 0 = null = undefined, mens for ?? er 0 != null/undefined) https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or